Sources/FoundationBenchmarkSupport/FoundationBenchmarkSupport.swift

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import FoundationEssentials
+
+public typealias FoundationEssentialsStringEncoding = String.Encoding
```

Comment:
I think this target should go under `Benchmarks` so you won't have to export it as a library.

Reply:
That does work too! (done)